modules/Hoot/tools/folderTree.js

```diff
@@ -727,7 +737,8 @@ export default class FolderTree extends EventEmitter {
                     d.children     = null;
                     data.selected  = false;
                 }
-            } else {
+            }
+            else {
```

Comment:
nitpick: I don't really care for this syntax using two lines

Reply:
fixed